charts/sonarqube-dce/templates/sonarqube-application.yaml

```diff
@@ -64,19 +64,19 @@ spec:
       {{- end }}
       {{- if .Values.postgresql.enabled }}
         - name: "wait-for-db"
-          image: {{ default "busybox:1.36" .Values.initContainers.image }}
+          image: {{ default (include "sonarqube.image" $) .Values.initContainers.image }}
           imagePullPolicy: {{ .Values.ApplicationNodes.image.pullPolicy  }}
           {{- if $securityContext := .Values.initContainers.securityContext }}
           securityContext:
 {{ toYaml $securityContext | indent 12 }}
           {{- end }}
           resources:
 {{ toYaml .Values.initContainers.resources | indent 12 }}
-          command: ["/bin/sh", "-c", "for i in $(seq 1 200); do nc -z -w3 {{ .Release.Name}}-postgresql 5432 && exit 0 || sleep 2; done; exit 1"]
+          command: ["/bin/sh", "-c", "for i in $(seq 1 200); do (echo >/dev/tcp/{{ .Release.Name}}-postgresql/5432) &>/dev/null && exit 0 || sleep 2; done; exit 1"]
```

Comment:
By killing entierely the postgresql container, the wait-for-db still complete successfully.